src/lib_ccx/hardsubx_russian.c

```diff
@@ -0,0 +1,995 @@
+#include "lib_ccx.h"
+#include "utility.h"
+
+#ifdef ENABLE_HARDSUBX
+//TODO: Correct FFMpeg integration
+#include <libavcodec/avcodec.h>
+#include <libavformat/avformat.h>
+#include <libavutil/imgutils.h>
+#include <libswscale/swscale.h>
+#include "allheaders.h"
+#include "hardsubx.h"
+#include "capi.h"
+
+//Global Flags
```

Comment:
Instead of being globals, can these not be made part of a separate Russian context? Helps to keep track too.

Reply:
I've tried that, but some values just need to be initialized and changed in more than one function and there are a lot of loops that changes the value. I first tried to do that but it was not possible